src/main/java/com/jcraft/jsch/DHECN.java

```diff
@@ -96,8 +96,7 @@ public void init(Session session,
     session.write(packet);
 
     if(JSch.getLogger().isEnabled(Logger.INFO)){
-      JSch.getLogger().log(Logger.INFO, 
-                           "SSH_MSG_KEX_ECDH_INIT sent");
+            JSch.getLogger().log( Logger.INFO, "SSH_MSG_KEX_ECDH_INIT sent with EC nistp" + key_size );
```

Comment:
Do we want to amend DHXEC in a similar manner to provide and indication which curve was sent?

Reply:
In fact, I'm wondering if it might be best to simply tweak all the DH* classes to just include the kex name?

Reply:
My experience of working with a different servers: often in their EC implementations something is broken; it helps a lot finding connection errors when you know on what curve we did agree.